assets/src/i18n/index.js

```diff
@@ -1,15 +1,17 @@
 import dictionary from './dictionary.js';
 
+function getObjProperty(object, propertyName) {
```

Comment:
could you please add a method description so that everyone understands how this method works? Thanks :)